submissions/asaMitaka/FriendsApp/script.js

```diff
@@ -0,0 +1,177 @@
+const url = 'https://randomuser.me/api/?results=12';
+
+let aToZLast = document.querySelector('.sortedAtoZLast');
```

Comment:
You almost exclusively use `let` variables while most of them should be `const`. Please review your code and change them.